instance/management/commands/instance_statistics_csv.py

```diff
@@ -131,7 +137,10 @@ def handle(self, *args, **options):
             ))
             sys.exit(1)
 
-        self.collect_instance_statistics(out, domains, start_date, end_date)
+        if options['users']:
+            self.collect_active_users(out, domains, start_date, end_date)
+        else:
+            self.collect_instance_statistics(out, domains, start_date, end_date)
```

Comment:
@janimo, we should add the `active users` data to the existing `collect_instance_statistics` function, i.e., the number of active users should be another column added to the existing report.

Reply:
@lgp171188 the existing task only produces stats for the trial instances created in the last month. Do you mean I should create another CSV which contains not only the active users but all the existing stats for all instances?

Reply:
@janimo, I hope my comments above answers your questions.

The current implementation of the management command produces a report with N columns and the periodic task primarily determines the list of instances to pass to that command based on some criteria (instances created in the last completed month). The desired result for this PR should be to add a new column with the number of active users in the output of the management command - i.e., it should now have N+1 columns.

With this done, the existing periodic task will automatically get this new column.

You will then have to implement another periodic task which just determines the list of instances to pass to the management command using different criteria - all the betatest instances, which includes the trial instances and the P&T plan subscribers. For the sake of simplicity, you can query all the instances that have an associated betatestapplication instance attached.